services/worker/src/worker/dtos.py

```diff
@@ -251,6 +251,12 @@ class DatasetModalitiesResponse(TypedDict):
     modalities: list[DatasetModality]
 
 
+class DatasetCroissantCrumbsResponse(TypedDict):
+    croissant_crumbs: Mapping[str, Any]
+    truncated_configs: bool
+    partial: bool
```

Comment:
The front-end expects the `croissant_crumbs` directly instead of this nested object no ?

Reply:
yes, you're right! I'll fix it

Reply:
should be fixed in https://github.com/huggingface/datasets-server/pull/2524/commits/6f33c4ae11cb334b0a844aa6427b664a633b6d7b